variables.tf

```diff
@@ -743,6 +743,22 @@ variable "efs_file_system_token" {
   default     = ""
 }
 
+variable "efs_throughput_mode" {
+  description = "(Optional) Throughput mode for the file system. Defaults to bursting. Valid values: bursting, provisioned, or elastic. When using provisioned, also set provisioned_throughput_in_mibps."
+  type        = string
+  default     = "bursting"
+  validation {
```

Comment:
Please remove the validation

Reply:
Why remove the validation? There is another validation in this file.

Reply:
its doubly redundant - the provider already checks that only select values are required and as new modes are added, they will be added at the provider without any code changes required here

Reply:
Done, thanks for the response.